CodeEdit/Features/Documents/WorkspaceDocument.swift

```diff
@@ -33,14 +33,19 @@ import Combine
         didSet { workspaceFileManager?.onRefresh() }
     }
 
-    var statusBarModel = StatusBarViewModel()
+    var debugAreaModel = DebugAreaViewModel()
     var searchState: SearchState?
     var quickOpenViewModel: QuickOpenViewModel?
     var commandsPaletteState: CommandPaletteViewModel?
     var listenerModel: WorkspaceNotificationModel = .init()
 
+    var anyCancellable: AnyCancellable?
+
     override init() {
         super.init()
+        anyCancellable = debugAreaModel.objectWillChange.sink { (_) in
+            self.objectWillChange.send()
+        }
```

Comment:
What does this code do? It gives a runtime warning

Reply:
Without it, when variables change, it does not recompute the view. For example, if it is removed, and the collapsed state is changed, it doesn’t open until doing something else that will cause the view to update.

Reply:
I pushed some changes which remove this code, and add the `DebugAreaViewModel` instance to the environment. This fixes the issues, because we can now observe that class. Previously, we observed the `WorkspaceDocument` instance, which is why we weren't getting any updates when something changed in the debug model.
This change will also be a bit more efficient, as we're only triggering a change in the debug instance, rather than the whole workspacedocument instance.